stdlib/pathlib.pyi

```diff
@@ -22,7 +23,11 @@ if sys.version_info >= (3, 9):
 
 __all__ = ["PurePath", "PurePosixPath", "PureWindowsPath", "Path", "PosixPath", "WindowsPath"]
 
+if sys.version_info >= (3, 13):
+    __all__ += ["UnsupportedOperation"]
+
 class PurePath(PathLike[str]):
+    parser: types.ModuleType
```

Comment:
Not sure if the `parser` type can be improved with a class that implement the BaseParser methods: https://github.com/python/cpython/blob/f1302c1e4785871197048d3701a5ecf4ed145596/Lib/pathlib/_abc.py#L55
mypy expects `posixpath` or `ntpath` at runtime so I couldn't find a way to make it work.

Reply:
It's only available on Python 3.13+ whatever the case, right?

```suggestion
    if sys.version_info >= (3, 13):
        parser: types.ModuleType
```

Reply:
Yes

Reply:
I think it's fine for the type just to be `types.ModuleType`. The main use is, I believe, so that you can do tests such as `if Path.parser is ntpath` or `if Path.parser is posixpath`, and reliably determine which backend `pathlib` is using on a given machine.